### PR TITLE
Feature/df 518 Admin - User Info

### PR DESCRIPTION
**Pull request by emina-str**
Some prt screens.
no info
with info

### CODE REVIEW

File: alv-portal-ui/src/app/admin/user-info/user-info.component.ts [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+    return this.userRoles == null || this.userRoles.length < 1;
+  }
+
+  private setActions(): void {
+    this.isUserInfoFilled = !!this.user || !this.isUserRoleEmpty();
+    this.isOnlyEIAMRoles = this.user == null && !this.isUserRoleEmpty() && this.userRoles.includes('ALLOW');
+  }
+
+  private confirmUnregister(): boolean {
+    return window.confirm(this.confirmMessage);
```
Comment (pavax) [inline alv-portal-ui/src/app/admin/user-info/user-info.component.ts]: use our confirm dialog

Comment (emina-str) [reply alv-portal-ui/src/app/admin/user-info/user-info.component.ts]: Implemented the modal confirm dialog.

File: alv-portal-ui/src/app/admin/user-info/user-info.component.ts [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+  private isUserRoleEmpty(): boolean {
+    return this.userRoles == null || this.userRoles.length < 1;
+  }
+
+  private setActions(): void {
+    this.isUserInfoFilled = !!this.user || !this.isUserRoleEmpty();
+    this.isOnlyEIAMRoles = this.user == null && !this.isUserRoleEmpty() && this.userRoles.includes('ALLOW');
+  }
+
+  private getRoleParam(): string {
```
Comment (pavax) [inline alv-portal-ui/src/app/admin/user-info/user-info.component.ts]: rename this method to: determineRoleToBeRemoved()

File: alv-portal-ui/src/app/admin/user-info/user-info.component.ts [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+  }
+
+  dismissAlert() {
+    this.alert = null;
+  }
+
+  onUnregister(): void {
+    this.modalService.openConfirm({
+      title: 'portal.admin.user-info.actions.unregister.title',
+      textHtml: this.confirmMessage,
```
Comment (pavax) [inline alv-portal-ui/src/app/admin/user-info/user-info.component.ts]: cant we just modifiy the open-confirm dialog to accept a contentKey and internally it uses the translation pipe?

File: alv-portal-ui/src/app/admin/user-info/user-info.component.ts (hunk 2 above)
Comment (emina-str) [reply alv-portal-ui/src/app/admin/user-info/user-info.component.ts]: Renamed, and Fabian said that it's ok as is (regarding HttpParams).

File: alv-portal-ui/src/app/admin/user-info/user-info.component.ts (hunk 3 above)
Comment (emina-str) [reply alv-portal-ui/src/app/admin/user-info/user-info.component.ts]: I've renamed to `content` and I'm passing only labelKey, but as I have to pass the `email` value to translation, I've added also the `contentParams` to confirm modal dialog. I'm not sure is it OK like that, or did you have something else in mind ... Can you please check?

File: alv-portal-ui/src/app/admin/user-info/user-info.component.html [hunk 4]
```diff
@@ -0,0 +1,155 @@
+<div class="container pt-md-4 pt-3 px-3">
+
+  <div class="alv-panel mb-3">
+    <h2 class="panel-title">
+      <i class="panel-icon fas fa-user"></i>
+      {{ 'portal.admin.user-info.menu-entry' | translate }}
+    </h2>
+    <form (validSubmit)="onSubmit()"
+          [formGroup]="form"
+          autocomplete="off"
```
Comment (pavax) [inline alv-portal-ui/src/app/admin/user-info/user-info.component.html]: autocomplete="off" is not needed if you use the alvFormSubmitValidation directive

File: alv-portal-ui/src/app/shared/backend-services/user-info/user-info-repository.ts [hunk 5]
```diff
@@ -25,4 +25,28 @@ export class UserInfoRepository {
     return this.http.post<void>(`${UserInfoRepository.USER_INFO_URL}${userId}/company-contact-templates`, companyContactTemplate);
   }
 
+  public loadUserByEmail(email: string): Observable<any> {
+    const params = new HttpParams().set('eMail', email);
+    return this.http.get(UserInfoRepository.USER_INFO_URL, {
```
Comment (pavax) [inline alv-portal-ui/src/app/shared/backend-services/user-info/user-info-repository.ts]: the userInfoRepository should directly return the response.body -> there is no need for a observe: 'response' (please change in all the methods

File: alv-portal-ui/src/app/admin/user-info/user-info.component.html (hunk 4 above)
Comment (emina-str) [reply alv-portal-ui/src/app/admin/user-info/user-info.component.html]: removed.

File: alv-portal-ui/src/app/shared/backend-services/user-info/user-info-repository.ts (hunk 5 above)
Comment (emina-str) [reply alv-portal-ui/src/app/shared/backend-services/user-info/user-info-repository.ts]: Removed.

**Pull request merged**